### PR TITLE
migrate to use mkr.CheckReports instead of monitoringChecksPayload

### PR DESCRIPTION
**Pull request by lufia**
I replaced mackerel.monitoringChecksPayload with mkr.CheckReports.

### CODE REVIEW

File: mackerel/check.go [hunk 1]
```diff
@@ -44,17 +19,31 @@ func (api *API) ReportCheckMonitors(hostID string, reports []*checks.Report) err
...
 			Name:                 report.Name,
-			Status:               report.Status,
+			Status:               mkr.CheckStatus(report.Status),
 			Message:              msg,
-			OccurredAt:           Time(report.OccurredAt),
-			NotificationInterval: report.NotificationInterval,
-			MaxCheckAttempts:     report.MaxCheckAttempts,
+			OccurredAt:           report.OccurredAt.Unix(),
+			NotificationInterval: normalize(report.NotificationInterval, NotificationIntervalFallback),
+			MaxCheckAttempts:     normalize(report.MaxCheckAttempts, 0),
```
Comment (susisu) [inline mackerel/check.go]: The minimum value of `maxCheckAttempts` is `1`.
https://mackerel.io/api-docs/entry/check-monitoring#post
(`0` will be omitted from JSON output and the default value `1` will be used in such a case, so there are no problems, though)

File: mackerel/check.go [hunk 2]
```diff
@@ -1,41 +1,16 @@
...
-}
-
-// MarshalJSON implements json.Marshaler.
-func (h monitorTargetHost) MarshalJSON() ([]byte, error) {
-	return json.Marshal(map[string]string{
-		"type":   "host",
-		"hostId": h.HostID,
-	})
-}
+const NotificationIntervalFallback = 10
```
Comment (susisu) [inline mackerel/check.go]: `make lint` fails. Please add some comments.

Comment (lufia) [reply mackerel/check.go]: sorry, I resolved it.

**Comment by lufia** [comment]
Thank you.

**Pull request merged**